packages/myst-frontmatter/src/licenses/licenses.yml

```diff
@@ -280,6 +280,53 @@ cases:
         content:
           url: https://example.com
     warnings: 1
+  - title: url string coerces to license from spdx list
+    raw:
+      license: https://creativecommons.org/licenses/by/4.0/
```

Comment:
Previously, this test case passed - i.e. setting `license` to a known SPDX url would validate to the correct SPDX license object.